google/cloud/internal/oauth2_compute_engine_credentials.cc

```diff
@@ -73,7 +141,13 @@ ServiceAccountMetadata ParseMetadataServerResponse(std::string const& payload) {
     if (!body.contains("email") || !body["email"].is_string()) return {};
     return body.value("email", "");
   };
-  return ServiceAccountMetadata{scopes(), email()};
+  auto universe_domain = [&]() -> std::string {
+    if (!body.contains("universe_domain") ||
+        !body["universe_domain"].is_string())
+      return {};
```

Comment:
This needs a brace around it.  Consider using an iterator to avoid making two lookups for `contains` and `operator[]`.